ReleaseNotes.md

```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## New in Release 2.11.0
+
+Post-training Quantization:
+
+- Features:
+  - (OpenVINO) Added Scale Estimation algorithm for 4-bit data-aware weights compression. The optional `scale_estimation` parameter was introduced to `nncf.compress_weights()` and can be used to minimize accuracy degradation of compressed models (note that this algorithm increases the compression time).
+  - (OpenVINO) Added GPTQ algorithm for 8/4-bit data-aware weights compression, supporting INT8, INT4, and NF4 data types. The optional `gptq` parameter was introduced to `nncf.compress_weights()` to enable the GPTQ algorithm.
+  - (OpenVINO) Added support for BF16 weights compression in `nncf.compress_weights()`.
```

Comment:
Let's reformulate it to avoid any confusion (one can think that we compress models to bf16):
Added support for models with BF16 weights in the weights compression method, `nncf.compress_weights()`.

Reply:
Done, thanks.